src/lighthouse.php

```diff
@@ -91,19 +91,9 @@
         'enable' => env('LIGHTHOUSE_CACHE_ENABLE', env('APP_ENV') !== 'local'),
 
         /*
-         * The name of the cache item for the schema cache.
+         * File path to store the lighthouse schema.
          */
-        'key' => env('LIGHTHOUSE_CACHE_KEY', 'lighthouse-schema'),
-
-        /*
-         * Allows using a specific cache store, uses the app's default if set to null.
-         */
-        'store' => env('LIGHTHOUSE_CACHE_STORE', null),
-
-        /*
-         * Duration in seconds the schema should remain cached, null means forever.
-         */
-        'ttl' => env('LIGHTHOUSE_CACHE_TTL', null),
+        'path' => base_path('bootstrap/cache/lighthouse-schema.php'),
```

Comment:
The reason I did this is because bootstrap cache paths are customizable in laravel as well https://github.com/laravel/framework/blob/a654897ad7f97aea9d7ef292803939798c4a02a4/src/Illuminate/Foundation/Application.php#L1065

Reply:
I don't see how they are customizable, everything from the base path onwards seems to be fixed.

Reply:
Thanks for digging into this. It makes sense to me now to have the option, I just wanted to see if we could cut down on complexity here if there is no actual use case.